OpenChange/MAPIStoreContactsMessage.m

```diff
@@ -170,6 +170,23 @@ - (int) getPidTagTitle: (void **) data
   return MAPISTORE_SUCCESS;
 }
 
+- (int) getPidTagProfession: (void **) data
+                 inMemCtx: (TALLOC_CTX *) memCtx
+{
+  NSString *stringValue;
+  int rc = MAPISTORE_SUCCESS;
+
+  stringValue = [[sogoObject vCard] role];
+  if (stringValue)
+    {
```

Comment:
Curly braces are unnecessary here. For uniformity, I'd use it without curly braces when a single statement appears.

It applies to all getters.